flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/MesosResourceManager.java

```diff
@@ -185,6 +190,8 @@ public MesosResourceManager(
 		this.workersInNew = new HashMap<>(8);
 		this.workersInLaunch = new HashMap<>(8);
 		this.workersBeingReturned = new HashMap<>(8);
+
+		this.webUiUrl = webUiUrl;
```

Comment:
I would move this line below this assignment:
```
this.taskManagerContainerSpec = Preconditions.checkNotNull(taskManagerContainerSpec);
```
so that the assignments of all constructor arguments are grouped.